docker-compose-base.yaml

```diff
@@ -0,0 +1,53 @@
+services:
+  media-nginx:
```

Comment:
if I understand correctly, https://github.com/MiSArch/frontend/pull/53 adds all the nginx conf for minio directly to the frontend nginx conf
if this service is only required in the dev docker compose file of this service, please move it directly there